assets/styl/_settings.styl

```diff
@@ -193,7 +193,117 @@ $colors ?= {
+//    'modules-alerts' : '_modules/_alerts',
+//    'modules-teasers' : '_modules/_teasers',
+//    'theme-skin' : '_theme/_skin'
+// }
+//
+// @require `colette.styl` /* will only '@require' files listed in '$css-main */
+//
+// /* my custom styles */
+// ```
+//
+//
+// **Usage example of `$css-critical`**
+//
+// You need to create a specific stylus file to handle your critical css.
+// ```
+// /* my_criticalcss_file.styl */
+//
+// $css-critical = {
+//    'modules-avatar' : '_modules/_avatar',
+//    'modules-breadcrumb' : '_modules/_breadcrumb',
+//    'modules-buttons' : '_modules/_buttons',
+//    'modules-socialbar' : '_modules/_socialbar',
+//    'modules-tabs' : '_modules/_tabs',
+// }
+//
+// /* @require files */
+// for name, file in $css-critical
+//     @require file
+//
+// /* => will compile to my_criticalcss_file.css */
+// ```
+//
+// Then you'll need to compile the css and inline it into your html markup.
+// Suppose you have a gulpfile in your project, and work with Twig templates,
+// you can create a `gulp` task to compile the css:
+//
+// ```
+// /* gulpfile.js */
+// gulp.task('criticalcss', function () {
+//    return gulp.src(cfg.cssDir + 'loop.styl')
+//    .pipe(stylus())
+//    .pipe(gulp.dest('css/'))
+// })
+// ```
+//
+// and then simply include the compiled css file into your template
+// (dont't forget the `<style>` tag aroud your `include`!):
+// ```
+// <style>
+//    {%  include 'css/my_criticalcss_file.css' %}
+// </style>
+// ```
+//
+// Styleguide: Settings.CssCollections
+$css-main ?= {
+    'mixins-contrast-check' : '_mixins/_contrastCheck'
+    'mixins-mask' : '_mixins/_mask'
+    'helpers-text' : '_helpers/_text'
+    'helpers-block' : '_helpers/_block'
+    'helpers-borders' : '_helpers/_borders'
+    'helpers-box' : '_helpers/_box'
+    'helpers-colors' : '_helpers/_colors'
+    'helpers-icons' : '_helpers/_icons'
+    'helpers-mask' : '_helpers/_mask'
+    'helpers-spaces' : '_helpers/_spaces'
+    'base-fonts' : '_base/_fonts'
+    'base-form-elements' : '_base/_form-elements'
+    'base-normalize' : '_base/_normalize'
+    'base-symbols' : '_base/_symbols'
+    'base-typography' : '_base/_typography'
+    'layout-footer' : '_layout/_footer'
+    'layout-grid' : '_layout/_grid'
+    'modules-alerts' : '_modules/_alerts'
+    'modules-avatar' : '_modules/_avatar'
+    'modules-breadcrumb' : '_modules/_breadcrumb'
+    'modules-buttons' : '_modules/_buttons'
+    'modules-form-single-row' : '_modules/_form-single-row'
+    'modules-form-widget' : '_modules/_form-widget'
+    'modules-icon' : '_modules/_icon'
+    'modules-loading' : '_modules/_loading'
+    'modules-media' : '_modules/_media'
+    'modules-navbar' : '_modules/_navbar'
+    'modules-required' : '_modules/_required'
+    'modules-socialbar' : '_modules/_socialbar'
+    'modules-tabs' : '_modules/_tabs'
+    'modules-teasers' : '_modules/_teasers'
+    'theme-skin' : '_theme/_skin'
+}
+$css-critical ?= {}
```

Comment:
`$css-main` seems enough, maybe rename as `$require`.
`$css-critical` could be removed.